### ISSUE TITLE
Can this work achieve fast channel hopping for channel splice?

### ISSUE THREAD

**Issue by hanzijun** [seemoo-lab/nexmon_csi#81]
Just like Intel 5300 or Atheros series have done. 
Although 80MHz is wider than 20 MHz, it is not enough for more fine-grained activities recognition tasks and power delay profile derivation. Can this work achieve fast channel hopping for channel splice?

**Comment by matthiasseemoo** [comment]
Yes you can perform the hopping. Simply change the channel on both sender and receiver after sending resp. receiving a frame from the expected sender.
